cmd/unpack/main.go

```diff
@@ -24,36 +39,99 @@ func main() {
 				fmt.Printf("Git commit: %s\n", version.String())
 				os.Exit(0)
 			}
-			bundleFS := os.DirFS(bundleDir)
-			bundleContents := map[string][]byte{}
-			if err := fs.WalkDir(bundleFS, ".", func(path string, d fs.DirEntry, err error) error {
-				if err != nil {
-					return err
-				}
-				if d.IsDir() {
-					return nil
-				}
-				data, err := fs.ReadFile(bundleFS, path)
-				if err != nil {
-					return fmt.Errorf("read file %q: %w", path, err)
-				}
-				bundleContents[path] = data
-				return nil
-			}); err != nil {
-				log.Fatalf("walk bundle dir %q: %v", bundleDir, err)
+			contentHandler := serveBundle(os.DirFS(bundleDir))
+			h := mux.NewRouter()
+			h.Handle("/content", contentHandler).Methods(http.MethodGet)
+			srv := http.Server{
+				Addr:         listenAddr,
+				Handler:      handlers.CombinedLoggingHandler(os.Stdout, h),
+				ReadTimeout:  1 * time.Second,
+				WriteTimeout: 10 * time.Second,
```

Comment:
Does the server expect a POST/write response?

Reply:
The server only accepts `GET` requests to `/content` and responds with the tar.gz of the bundle dir. That's the only possible interaction with the server.

Reply:
Ok, yes I think I didn't word that correctly -- what I meant to say was, since the server only supports GET requests, should we have a write timeout set at all? Is it being used somewhere?

Reply:
Ah. As I understand it, ReadTimeout is the timeout for the server reading the request body. WriteTimeout is the timeout for the server writing the response back to the client.

I think its a best practice to set both timeouts to avoid issues with clients that open a connection and then don't send the request body or read the response body.

For an individual bundle, I would think 10 seconds is enough time to send the tar.gz, but I have nothing concrete to back that up. I figure its best to err on the low side and we can always increase it later if we find that 10 seconds is insufficient.